apps/docs/components/ui/aside/aside.css

```diff
@@ -23,10 +23,10 @@
     position: absolute;
     content: "";
     display: block;
-    width: 1px;
+    width: var(--hd-border-size);
```

Comment:
should we name our local variables with the component name prefix to avoid clashing with global variables?.